### ISSUE TITLE
Engineer User Story

### ISSUE THREAD

**Issue by CameronCrutcher** [RhoAl/CS46X-Engineering-Simulations-4-#3]
User Story Statement: As an engineer, I want to import a hill into a simulation, so that I can see the forces acting on a car that will be driving up it.

Background: We want to accomplish this task so we can see a real-world example of forces acting upon a car, possibly to make sure a car is safe, or efficient. We will have the methods to accomplish importing the hill into the simulation, but we are not sure if we will be able to provide a detailed description of the forces acting on the car.

Solutions/Requirements: 
Functional Requirements:
- Import a mesh into blender
- Have the car interact with the mesh properly
- Be able to see the forces acting on the car at a certain moment of time

Nonfunctional Requirements:
- The simulation must reliably mimic real world physics
- Performance must have a steady framerate of 30 fps

Dependencies:
- Workflow pipeline needs to be finalized
- Integrating meshes from blender to bevy
- Car collision with imported meshes implemented

Tasks:
- Retrieve mesh from photogrammetry pictures (Sub Tasks: Retrieving pictures, turning pictures into mesh)
- Integrate mesh into simulation (blocked by previous task)
- Get collision between car and mesh to function (blocked by previous two tasks)
- Data export of the forces acting on the car (Possible Stretch goal for now)

Time Estimate:
Each task should be able to be completed in a week, so this whole user story could be about 3 weeks if we can work or the data export at the same time that we are working on creating terrain.

Acceptance Criteria:
We have a simulation that provides an accurate portrayal of real world physics on terrain that we have generated and imported in.
